pkg/webhooks/utils/utils.go

```diff
@@ -38,6 +39,16 @@ func SliceContains(needle string, haystack []string) bool {
 	return false
 }
 
+func RegexSliceContains(needle string, haystack []string) bool {
```

Comment:
Is the Regex required anymore if the list is just fixed strings? Changing to a simple string lookup could eek out a bit more performance.

Reply:
I think this is blocked by the above comment. Definitely open to removing this function if we can figure out a way to unblock

Reply:
Yes, disregard as I didn't see the `redhat-*` wildcard regex.